tests/test_installer.py

```diff
@@ -149,7 +149,7 @@ def _specifier(version):
             "4.10.3-7",
             {
                 "conda": "4.10.3",
-                "mamba": ">=1.4.1",
+                "mamba": ">=1.1.0",
```

Comment:
This was installed fresh but still got stuck at 1.1.0, I guess it was held back for some reason. I find digging into why a conda-forge package is held back during install/upgrade is soo tricky, and I failed with my preliminary investigations looking into the mamba recipie for example.

Reply:
> it may not upgrade all the way to 1.4 if other packages are pinned in the base env

:+1: